docs/application/native/get-started/overview.md

```diff
@@ -1,22 +1,24 @@
 # Get Started with Native Applications
 
-When creating your first Tizen native application, you can select from the following options, which all use the C language:
+Tizen Native applications are developed in C. You can use the available native API modules to have advanced system settings and additional functionalities.
```

Comment:
I understood what you said. I guess you'd like to describe Tizen Native application first, but your explanation is a little awkward.

I'd like to suggest as follows:
```suggestion
You can create your Tizen application  with Tizen native APIs, which are written in C language and provide Tizen additional functionalities.
```

If you want to add the definition of Tizen native application, please don't hesitate.

Reply:
Thank you for understanding. I have taken your suggestion and tweaked a little.
Can we rephrase as:

Reply:
Can I add something here?
The first sentence is an introduction and a key sentence.  
Some suggested sentences are the first sentence on the linked pages starting from "Create...".
We need a change. 

Are there better sentences that introduce Native, Web, .NET applications respectively?
What a Tizen XXX App is and developers can do what with it.